src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -105,6 +106,11 @@ define(function (require, exports, module) {
     function start(){
         window.addEventListener("message", _listener);
 
+        var autoUpdate = BrambleStartupState.ui("autoUpdate");
+        if(typeof autoUpdate === "boolean") {
```

Comment:
it feels odd to have to type check a value that we supposedly control. Would we at least be able to enforce that whetever it's set it is a boolean? Then we don't have to bother with the check here, and we can let the handler function do whatever it needs to do when it gets "not a bool" pass into it.